src/main/java/org/jahia/modules/contenteditor/api/forms/impl/EditorFormServiceImpl.java

```diff
@@ -94,6 +94,10 @@
 
     // we extend the map from SelectorType.defaultSelectors to add more.
     private static Map<Integer, Integer> defaultSelectors = new HashMap<>();
+    // Regex for range format of a constraint value, extracted from org.apache.jackrabbit.spi.commons.nodetype.constraint.NumericConstraint
+    private static final Pattern RANGE_PATTERN = Pattern.compile("([\\(\\[]) *(\\-?\\d+\\.?\\d*)? *, *(\\-?\\d+\\.?\\d*)? *([\\)\\]])");
+    private static final int LOWER_LIMIT_RANGE_IDX = 2;
+    private static final int UPPER_LIMIT_RANGE_IDX = 3;
```

Comment:
UPPER_LIMIT_RANGE_IDX seems to not been used

Reply:
you're more than right, thank you for the review